flink-contrib/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBKeyedStateBackend.java

```diff
@@ -834,7 +836,7 @@ private void restoreKVStateData() throws IOException, RocksDBException {
 	}
 
 	@Override
-	protected <N, T> InternalValueState<N, T> createValueState(
+	public <N, T> InternalValueState<N, T> createValueState(
```

Comment:
I think these should stay `private`/`protected`. It was mistake to make them `public` in the other state backends. Creating a state can always be achieved with `getPartitionedState()` and a `StateDescriptor`.